build/x86_64_centos6/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:6
+
+ENV PATH ${PATH}:/usr/local/bin
+
+RUN yum update -y && yum install -y \
+  autoconf \
```

Comment:
Let's indent the packages, the same way they are indented in the Debian `Dockerfile`. That makes it immediately obvious where the next command starts.

Reply:
Done.